terratest/src/test/pega/data/values_with_overidden_liveness_probe_config.yaml

```diff
@@ -0,0 +1,375 @@
+      # cpuRequest: 2
+      # memRequest: "12Gi"
+      # cpuLimit: 4
+      # memLimit: "12Gi"
+      # initialHeap: "4096m"
+      # maxHeap: "8192m"
+
+      # To configure an alternative user for custom image, set value for runAsUser.
+      # See, https://github.com/pegasystems/pega-helm-charts/blob/master/charts/pega/README.md#security-context
+      # securityContext:
+      #   runAsUser: 9001
+
+      hpa:
+        enabled: true
+
+      # Set enabled to true to include a Pod Disruption Budget for this tier.
+      # To enable this budget, specifiy either a pdb.minAvailable or pdb.maxUnavailable
+      # value and comment out the other parameter.
+      pdb:
+        enabled: false
+        minAvailable: 1
+        # maxUnavailable: "50%"
+
+    - name: "batch"
+      # Create a background tier for batch processing.  This tier uses
+      # a collection of background node types and will not be exposed to
+      # the load balancer.
+      nodeType: "BackgroundProcessing,Search,Batch,RealTime,Custom1,Custom2,Custom3,Custom4,Custom5,BIX"
+
+      replicas: 1
+      javaOpts: ""
+
+      pegaDiagnosticUser: ""
+      pegaDiagnosticPassword: ""
+
+      deploymentStrategy:
+        rollingUpdate:
+          maxSurge: 1
+          maxUnavailable: 0
+        type: RollingUpdate
+
+      livenessProbe:
+        port: 8081
+        failureThreshold: 5
+        periodSeconds: 100
+
+
+      # To configure an alternative user for your custom image, set value for runAsUser
+      # See, https://github.com/pegasystems/pega-helm-charts/blob/master/charts/pega/README.md#security-context
+      # securityContext:
+      #   runAsUser: 9001
+
+      hpa:
+        enabled: true
+
+      # Set enabled to true to include a Pod Disruption Budget for this tier.
+      # To enable this budget, specifiy either a pdb.minAvailable or pdb.maxUnavailable
+      # value and comment out the other parameter.
+      pdb:
+        enabled: false
+        minAvailable: 1
+        # maxUnavailable: "50%"
+
+    - name: "stream"
+      # Create a stream tier for queue processing.  This tier deploys
+      # as a stateful set to ensure durability of queued data. It may
+      # be optionally exposed to the load balancer.
+      nodeType: "Stream"
+
+      # Pega requestor specific properties
+      requestor:
+        # Inactivity time after which requestor is passivated
+        passivationTimeSec: 900
+
+      service:
+        port: 7003
+        targetPort: 7003
+
+      # If a nodeSelector is required for this or any tier, it may be specified here:
+      # nodeSelector:
+      #  disktype: ssd
+
+      ingress:
+        # Enter the domain name to access web nodes via a load balancer.
+        #  e.g. web.mypega.example.com
+        domain: "YOUR_STREAM_NODE_DOMAIN"
+        tls:
+          # Enable TLS encryption
+          enabled: true
+          # secretName:
+          # useManagedCertificate: false
+          # ssl_annotation:
+
```

Comment:
Test the case where livenessProbe.period seconds exceeds the max classloader retry timeout.